README.md

```diff
@@ -2,64 +2,38 @@
 Pi Video Machine - a scalable, synchronized, and networked-controlled, raspberry pi based, video player machine for video installations
 
 
-### { This is a work in progress }
-
-- TODO: Create another python file to control two display
-
-
 ## Requirements
 
 - [Python3](https://www.python.org/downloads/)
 - [Max/MSP](https://cycling74.com/)
-
+- [Raspberry Pi OS](https://downloads.raspberrypi.org/raspios_oldstable_armhf/images/raspios_oldstable_armhf-2022-09-26/2022-09-22-raspios-buster-armhf.img.xz)
+- [OMXPlayer](https://github.com/KaneBetter/omxplayer)
 
 ## Installation
 
-### Raspberry Pi OS
-The recommended Raspberry Pi OS version is:
-
-<img width="569" alt="image" src="https://user-images.githubusercontent.com/51472016/197014809-c39fc6c9-0a08-4b05-92cf-f79fead3c000.png">
-
-To install this to your Raspberry Pi, you need the following things:
-+ SD card reader.
-+ A laptop (windows/MacOSLinux).
+```mermaid
+graph TD;
+A[1. Install the required Raspberry Pi OS] --> B;
+B[2. Build and install customized OMXPlayer] --> C;
+C[3. Set up the environment for PVM]
+```
```

Comment:
ok that is cool.
Question: can you hyperlink each one to their section?

Only problem here is that... it is incomplete. It tells all about the RPI devices, but nothing about the control machine.

Reply:
I tested it and found that although I could add hyperlinks, it was difficult to click on Github, so we couldn't add hyperlinks.

Reply:
Add install MAX8 step